blocks/notification/notification.js

```diff
@@ -0,0 +1,50 @@
+import {
+  getCookie,
+  setCookie,
+  createStringToHash,
+} from '../../scripts/scripts.js';
+
+/**
+ * decorates the notification
+ * @param {Element} block The notification block element
+ */
+export default async function decorate(block) {
+  const outerDivElement = document.createElement('div');
+  outerDivElement.className = 'notification-outer-block';
```

Comment:
I would do a for each loop for the children element of the block so that it's clear and doesn't resolve in conflicts when there are multiple notifications. See below for an example.

```
export default function decorate(block) {
  [...block.children].forEach((element) => {
    element.classList.add('notification-block__inner');

    const outerDivElement = element.createElement('div');

  });
}
```

Reply:
@proeung As far as I remember, I had a discussion with you and @keith-kaplan before starting this component that we will not be supporting multiple notification, thats why I didn't looped the block content.

Reply:
@proeung The image used is coming from whatever is added to the document, I was not able to find the correct image that can be used for this component.
can you help me with that.